ui/shared/chart/ChartWidget.pw.tsx

```diff
@@ -25,7 +26,7 @@ const props: Props = {
   ],
   title: 'Native coin circulating supply',
   description: 'Amount of token circulating supply for the period',
-  units: 'ETH',
+  units: currencyUnits.ether,
```

Comment:
This data is returned by the Stats microservice and cannot be configured through the frontend app's environment variables. Therefore, this change is irrelevant.